ngx-tools/testing/README.md

```diff
@@ -238,14 +205,58 @@ beforeEach(() => {
 });
 ```
 
+### `TokenEscalatorMock`
+
+[[source]](src/mocks/token-escalator.mock.ts)
+
+TODO <!-- https://github.com/GetTerminus/ngx-tools/issues/317 -->
```

Comment:
@deanterm Once this PR is merged, I'd love if someone from Engage that has actually used these mocks created an example for this and the next mock